pocs/tests/utils/test_utils.py

```diff
@@ -51,3 +52,30 @@ def test_has_camera_ports():
 
     for port in ports:
         assert port.startswith('usb:')
+
+
+def test_countdown_timer_bad_input():
+    with pytest.raises(ValueError):
+        assert CountdownTimer('d')
+
+    with pytest.raises(ValueError):
+        assert CountdownTimer(current_time())
+
+    with pytest.raises(AssertionError):
+        assert CountdownTimer(-1)
+
+
+def test_countdown_timer_non_blocking():
```

Comment:
Could you add tests with float and u.second? For example:
```
for arg, expected_duration in [(2, 2.0), (0.5, 0.5), (1 * u.second, 1.0)]:
    timer = CountdownTimer(arg)
    assert timer.duration == expected_duration
...
```

Reply:
Done, thanks.